bin/alpen-bridge/src/rpc_server.rs

```diff
@@ -685,47 +714,22 @@ impl StrataBridgeMonitoringApiServer for BridgeRpc {
 
     async fn get_claims(&self) -> RpcResult<Vec<Txid>> {
         // Use the cached contracts
-        let all_entries = self.cached_contracts.read().await.clone();
+        let cache = self.cached_contracts.read().await.clone();
```

Comment:
I very strongly question whether cloning the entire cache in stack memory is a good idea.